frontend/apps/desktop/forge.config.js

```diff
@@ -7,12 +7,16 @@ const devProjectRoot = path.join(process.cwd(), '../../..')
 const LLVM_TRIPLES = {
   'darwin/x64': 'x86_64-apple-darwin',
   'darwin/arm64': 'aarch64-apple-darwin',
-  'windows/x64': 'x86_64-pc-windows-msvc',
+  'win32/x64': 'x86_64-pc-windows-msvc.exe',
   'linux/x64': 'x86_64-unknown-linux-gnu',
   'linux/arm64': 'aarch64-unknown-linux-gnu',
 }
 
 function getPlatformTriple() {
+  if (process.env.APP_TARGET == 'aarch64-apple-darwin') {
+    return process.env.APP_TARGET
```

Comment:
this isn't really a hack, we could switch to this behavior for the rest of the platforms 😃

Reply:
wdym?

Reply:
I changed it to get the daemon name for all platforms. let me know if now is better!